app/code/Magento/InventoryConfiguration/Model/StockConfigurationInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryConfiguration\Model;
+
+interface StockConfigurationInterface
+{
+    public function validate($sku, $stockId, $qtyWithReservation, $isSalable) : bool;
+}
```

Comment:
why does Configuration interface has method Validate?

Reply:
Apply? Check? What is correct naming?